requirements.txt

```diff
@@ -0,0 +1,2 @@
+autopep8
```

Comment:
I agree regarding the styling.  But won't a user be required to install pep8 to use the library?